### ISSUE TITLE
Light/Dark Favicon Logo

### ISSUE THREAD

**Issue by distresscode** [the-djmaze/snappymail#643]
The favicon is black and is visible if you're using light mode in the browser. However, it's hard to see with any dark theme/mode on. I suggest adding two logos and have it change depending on the system/browser theme. I'm not sure how github does it but changes from either a white or black favicon.

_It looks much darker depending on the angle you're sitting at._
![snappy](https://user-images.githubusercontent.com/2686608/200592707-3c113ce7-7ec0-4523-bb8c-e432f5b08710.png)

![snappy2](https://user-images.githubusercontent.com/2686608/200592698-a6d68f28-24cf-421b-968f-b214ef0eb0ca.png)

**Comment by the-djmaze** [comment]
![afbeelding](https://user-images.githubusercontent.com/3752035/200624593-8cc48db9-6995-42df-ae29-d7db0575b0f8.png)

https://www.billerickson.net/favicon-dark-mode/

**Comment by distresscode** [comment]
@the-djmaze what does it mean in layman's terms? don't let my username fool you.. I'm no coder :)

**Comment by the-djmaze** [comment]
It means that github has an svg.
So i added one to the code.

**Comment by distresscode** [comment]
Good to know. Thanks for the speedy update!
